src/components/StaffList/StaffList.scss

```diff
@@ -0,0 +1,8 @@
+.react-autosuggest__input {
+  width: 100%;
+}
+
+
+.react-autosuggest__suggestions-container--open {
+  border: 1px solid #aaa;
```

Comment:
Usually it's best to define a variable and then use that for the color (even if just in one place), so that other people have some idea what it should look like intuitively

Reply:
👍